src/firstboot/firstboot.c

```diff
@@ -323,15 +323,15 @@ static int prompt_locale(int rfd) {
 
                 print_welcome(rfd);
 
-                r = prompt_loop("Please enter system locale name or number",
+                r = prompt_loop("Please enter the new locale name or number",
```

Comment:
For locale, please keep 'system':
`Please enter the new system locale name or number`